Formula/macpine.rb

```diff
@@ -6,6 +6,20 @@ class Macpine < Formula
   license "Apache-2.0"
   head "https://github.com/beringresearch/macpine.git", branch: "main"
 
+  livecheck do
+    url :stable
+    regex(/^v?\.?(\d+(?:\.\d+)*)$/i)
+    strategy :git do |tags, regex|
+      tags.map do |tag|
+        version = tag[regex, 1]
+        next if version.blank?
+
+        # Naively convert tags like `v.01` to `0.1`
+        tag.match(/v\.\d+/i) ? version.chars.join(".") : version
```

Comment:
```suggestion
        tag.match(/^v\.?\d+$/i) ? version.chars.join(".") : version
```

Coming back to this, I should probably tighten up this regex just to be on the safe side.